common/lib/protocol-definitions/src/config.ts

```diff
@@ -34,4 +34,21 @@ export interface IClientConfiguration {
 
     // Summary algorithm configuration. This is sent to clients when they connect
     summary: ISummaryConfiguration;
+
+    /**
+     * noopTimeFrequency & noopCountFrequency control how often a client with "write" connection needs to send
+     * noop messages in case no other ops are being sent. Any op (including noops) result in client
```

Comment:
Worth pointing out that noops are sent only if client did not have a chance to communicate its reference sequence number via other ops